2022-01-12 10:11:10,613	ERROR import_thread.py:89 -- ImportThread: Connection closed by server.
[2022-01-12 10:21:09,809 E 145036 145244] gcs_server_address_updater.cc:76: Failed to receive the GCS address for 600 times without success. The worker will exit ungracefully. It is because GCS has died. It could be because there was an issue that kills GCS, such as high memory usage triggering OOM killer to kill GCS. Cluster will be highly likely unavailable if you see this log. Please check the log from gcs_server.err.
